MAUI/Scheduler/cell-selection-view.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: cell-selection
+title: Cell selection in the .NET MAUI Scheduler | Syncfusion
+description: Learn here all about how to customize the cell selection in Syncfusion .NET MAUI Scheduler (SfScheduler) control.
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Cell Selection View in .NET MAUI Event Scheduler (SfScheduler).
+
+The Scheduler provides the capability to customize the selection of month and timeslot cells using the `cellselectionview`.
+
+## Stroke
+
+The cell selection border color can be customized by using the `Stroke` property in the `CellSelectionView`.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="3" %}
+
+<scheduler:SfScheduler x:Name="Scheduler">
+    <scheduler:SfScheduler.CellSelectionView>
+            <scheduler:SchedulerCellSelectionView Stroke="Red"/>
+    </scheduler:SfScheduler.CellSelectionView>
+</scheduler:SfScheduler>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="2" %}
+this.Scheduler.CellSelectionView.Stroke = Brush.Green;
+{% endhighlight %}
+{% endtabs %}
+
+## Background
+
+The cell selection background color can be customized by using the `Background` property in the `CellSelectionView`.
+The default value is Brush.Transparent.
+
+Note: As default `stroke` will have value, if background alone required, you can set stroke property to Transparent.
```

Comment:
Note: As default, `stroke` will have value; if background alone is required, you can set the stroke property to transparent.

Reply:
I have completed the corrections